equivalences.csv

```diff
@@ -11,8 +11,8 @@ BEL,Adaptor Protein II Complex,Adaptor_protein_II
 BEL,Adaptor Protein III Complex,Adaptor_protein_III
 BEL,ADCY Family,ADCY
 BEL,ADH Family,ADH
-BEL,ADRA Family,ADRA
 BEL,ADRA2 Family,ADRA2
+BEL,ADRA Family,ADRA

```

Comment:
Many of the old entries were reordered in this file, let's try to revert the changes for existing entries and just add the new ones.